### ISSUE TITLE
Unable to deploy the github project to the specified directory

### ISSUE THREAD

**Issue by hubingkang** [Dokploy/dokploy#284]
### To Reproduce

It could be deployed normally before version 0.3.1, but after just upgrading to 0.5.0, an error is being reported. No deployment-related configurations were changed prior to this, and the corresponding directory exists on GitHub.

<img width="1307" alt="image" src="https://github.com/user-attachments/assets/07e132b1-5193-4ea1-8911-3541a4d5ba4f">

<img width="782" alt="image" src="https://github.com/user-attachments/assets/c4a86f22-2761-41af-aece-735403441acc">

### Current vs. Expected behavior

Hopefully, the directory can be identified for deployment

### Provide environment information

```bash
dolploy 0.5.0
```

### Which area(s) are affected? (Select all that apply)

Application

**Comment by Siumauricio** [comment]
@hubingkang yes before the 0.5.0 it automatically extract the context path but now you need to set the exact dockerfile name

**Comment by hubingkang** [comment]
@Siumauricio How do I set the Dockerfile path? I tried setting./ss/Dockerfile, /ss/Dockerfile,./ss, /ss all failed

**Comment by Siumauricio** [comment]
Can you share the structure of your project?

**Comment by hubingkang** [comment]
![image](https://github.com/user-attachments/assets/6c43b14c-eb74-4d63-876b-b5635315c3a4)

![image](https://github.com/user-attachments/assets/fac74eac-29e1-4ac4-9f22-97b5dd6f61ad)

**Comment by Siumauricio** [comment]
Ok, in the buildpath set /ss and in the dockefile path set Dockefile

**Comment by rcauler** [comment]
It seems to be using the Dockerfile path as the build path.

I'm putting / as the build path and /apps/my-app/Dockerfile as the Dockerfile path.

It's not finding files that should be in the root, like /pnpm-workspace.yaml

```
COPY pnpm-workspace.yaml pnpm-workspace.yaml
```

Using the command `docker build -t my-app . -f ./apps/my-app/Dockerfile` at the root of the repository works.

The deployment log in Dokploy shows this:

```
 > [install 2/6] COPY pnpm-workspace.yaml pnpm-workspace.yaml
------
Dockerfile:11
--------------------
   9 | FROM base AS install
  10 | WORKDIR /app
  11 | >>> COPY pnpm-workspace.yaml pnpm-workspace.yaml
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref f6af26c9-6cda-4a24-8aa5-343c781b5897::kfm64jfwoj6vdt1gwgnbyx7ms: "/pnpm-workspace.yaml pnpm-workspace.yaml": not found
```

**Comment by Siumauricio** [comment]
I understand @rcauler, you have a good point I think we should run the dockerfile in the root of all the code and only specify the path of the dockerfile, I will fix this today!

**Comment by Siumauricio** [comment]
@cyjme @rcauler I will add a new docker context path input to have more flexibility, because I was thinking to use the first folder we find but the cases could be different so in order to cover any case I will add a new input to handle this, is optional btw

**Comment by Siumauricio** [comment]
@rcauler @cyjme Released the v0.6.0 which contains this changes
